docfx.json

```diff
@@ -159,6 +159,7 @@
         "docs/azure/sdk/**/*.*": "https://github.com/azure/azure-sdk-for-net",
         "docs/fsharp/**/**.md": "https://github.com/dotnet/fsharp",
         "docs/fundamentals/code-analysis/quality-rules/**/*.md": "https://github.com/dotnet/roslyn-analyzers/issues",
```

Comment:
```suggestion
        "docs/fundamentals/code-analysis/quality-rules/**/ca*.md": "https://github.com/dotnet/roslyn-analyzers/issues",
```

Reply:
I'd leave it to catch other pages (e.g. index and category overview pages)